### PR TITLE
flatten helper

### PR DESCRIPTION
**Pull request by Asherlc**
Integration tests and documentation taken from https://github.com/DockYard/ember-composable-helpers/pull/96

### CODE REVIEW

File: addon/helpers/flatten.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+const {
+  Helper,
+  observer,
+  set,
+  isArray
+} = Ember;
+
+// lodash's flatten method doesn't work on Ember arrays
```
Comment (martndemus) [inline addon/helpers/flatten.js]: :fire: Kill this comment please :fire:

File: addon/helpers/flatten.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+const {
+  Helper,
+  observer,
+  set,
+  isArray
+} = Ember;
```
Comment (martndemus) [inline addon/helpers/flatten.js]: Use `import Helper from 'ember-helper'` etc.. please!

**Comment by martndemus** [comment]
Needs entry in `addon/index.js`

**Comment by Asherlc** [comment]
@martndemus  I believe this is all done.

**Comment by martndemus** [comment]
@homu r+

**Pull request merged**